.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,55 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ release/* ]
+#   pull_request:
```

Comment:
nit: if we're not planning on using it, let's remove this commented out code.

Reply:
Removed